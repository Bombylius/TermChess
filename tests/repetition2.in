g1 f3 b8 c6 f3 h4 c6 a5 h4 f3 a5 c6 f3 h4 c6 a5 h4 f3 .c a5 c6 f3 h4 c6 a5 .c 
