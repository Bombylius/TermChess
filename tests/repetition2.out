175921860444160
327680
4611686603885117440
43234885634
35184913154048
2216203386880
4611686603885117440
43234885634
35184913154048
2216203386880
4611686603885117440
43234885634
r1bqkbnr/pppppppp/8/n7/7N/8/PPPPPPPP/RNBQKB1R w KQkq - 12 7
4
